### DISCUSSION TITLE
Picocomputer emulator

### DISCUSSION THREAD

**Discussion by varomix** [picocomputer/community#131] [Q&A]
Hi
Are there plans to have an emulator for development? 
I want to program even when I go to bed or other parts of my house or even when I'm out I can't carry the picocomputer everywhere :)

Or how do you guys develop? only when you have the picocomputer by your side, any alternatives?

I could try doing something like and emulator but I see more clever people than me hang around here.

Thanks

**Comment by netzerohero** [comment]
See related cross-post [here](https://github.com/orgs/picocomputer/discussions/149#discussioncomment-14128645).
